docs/cloud-native-security/cspm-findings.asciidoc

```diff
@@ -37,7 +39,7 @@ You can filter findings data in two ways:
 [discrete]
 [[cspm-customize-the-findings-table]]
 === Customize the Findings table
-Use the toolbar buttons in the upper-left of the Findings table to customize the columns you want displayed:
+When grouping is disabled, you can use the toolbar buttons in the upper-left of the Findings table to select which columns appear:
```

Comment:
Is not necessarily when grouping is disabled, users can still customize the table in the grouped view as long the table is visible